cli/add_domain.sh

```diff
@@ -31,41 +31,51 @@ REAL_CNAME=$(dig +short -t CNAME "$DOMAIN")
 REAL_CNAME=${REAL_CNAME:-"None"} # Weird hack to get "None" printed instead of a blank line
 
 if [ "$NEEDED_CNAME" != "$REAL_CNAME" ]; then
-    echo "The domain $DOMAIN does not have the correct CNAME. Expected: $NEEDED_CNAME, Actual: $REAL_CNAME"   
-	exit 1 
+	echo "The domain $DOMAIN does not have the correct CNAME. Expected: $NEEDED_CNAME, Actual: $REAL_CNAME"
+	echo "Falling back to TXT record checking..."
+	NEST_DOMAIN_VERIFICATION=$(dig +short -t TXT nest_domain_verification.$DOMAIN | tr -d '"')
```

Comment:
Should it be `_nest_domain_verification` instead of `nest_domain_verification`? I feel like that would be helpful to distinguish the record from a real hostname.